Update communication-channels.md to add channel information (#3526)
the-turing-way/the-turing-way: book/website/community-handbook/communication-channels.md

- Discussion #3081 about [Technical Improvements to enhance the usability of The Turing Way Book (GSoC 2023)](https://github.com/the-turing-way/the-turing-way/discussions/3081)

To indicate the specific nature of the issue/discussion, we also use words such as `TALK`, `INVITATION` and `FEEDBACK NEEDED` in square brackets before writing the actual title.

### Slack Workspace

_The Turing Way_ Slack Workspace (theturingway.slack.com) is one of our core communication channels. 
This platform enables instant messaging, fostering quick exchanges and real-time collaborations among team members.
It is a space where folks ask questions, learn about each other’s work, and get involved with The Turing Way and other projects. 
While it is not the only communication channel used by the community, it is a popular platform, increasingly used by remote teams around the world.

You can join the workspace using the [joining invitation](https://theturingway.slack.com/join/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw#/shared-invite/email).

### Social media channels

> 
>  ## Additional Social Media Platforms

Author: ```suggestion
```

(ch-communication-platforms)=
# Communication Platforms

In _The Turing Way_, our [GitHub organisation](https://github.com/the-turing-way) serve as the primary source of information.
The main [project repository](https://github.com/the-turing-way/the-turing-way) hosts book chapters and project management documentation and resources.
In addition, the project delivery team makes use of various communication platforms to facilitate effective information dissemination within and beyond the engaged community.

Given the diverse range of formal and informal projects managed by both the project delivery team and community members, the communication within _The Turing Way_ community can be broadly categorised as synchronous and asynchronous.

## GitHub for Synchronous and Asynchonous Communications

[GitHub Issue](https://github.com/the-turing-way/the-turing-way/issues) and [Github discussions](https://github.com/the-turing-way/the-turing-way/discussions) 
serve as a dynamic space where updates, queries, and discussions happen. 

> Discussions, although not as frequently used by community members as the issues and Pull Requests, allow the project delivery team to share information related to specific topics, maintaining transparency among community members.

Author: ```suggestion
The GitHub 'discussions' feature, although not as frequently used by community members as the issues and Pull Requests, allow the project delivery team to share information related to specific topics, maintaining transparency among community members.
```

These avenues ensure that even individuals not actively participating in a conversation can stay informed about ongoing developments and contribute where they can. 
Contributors can respond in real-time, but the system is mainly used to accommodate varying time zones, allowing participants to engage at their convenience.

To streamline communication, issues that seek input or require voting are labelled appropriately, such as [`community`](https://github.com/the-turing-way/the-turing-way/labels/community), when a certain discussion requires community input.
When soliciting feedback, adding clear titles, mentioning the right people and indicating timelines help improve the engagement.
These issues can be crossposted on other communication channels to boost their visibility.

Some examples of when issues and discussions can be used for communication purposes are the following:

> - Issue #2811 about [Using/supporting open infrastructure - suggested alternatives to currently used tools](https://github.com/the-turing-way/the-turing-way/issues/2811)
> - Discussion #3081 about [Technical Improvements to enhance the usability of _The Turing Way_ Book (GSoC 2023)](https://github.com/the-turing-way/the-turing-way/discussions/3081)

Author: ```suggestion
- Inviting community inputs, ideas and feedback: Issue #2811 about [Using/supporting open infrastructure - suggested alternatives to currently used tools](https://github.com/the-turing-way/the-turing-way/issues/2811)
- Keeping a record of an ongoing project: Discussion #3081 about [Technical Improvements to enhance the usability of _The Turing Way_ Book (GSoC 2023)](https://github.com/the-turing-way/the-turing-way/discussions/3081)
```

> To indicate the specific nature of the issue/discussion, we also use words such as `TALK`, `INVITATION` and `FEEDBACK NEEDED` in square brackets before writing the actual title.

Author: ```suggestion
To indicate the specific nature of the issue/discussion, we also use words such as `[TALK]`, `[INVITATION]` and `[FEEDBACK NEEDED]` (in square brackets) before writing the full title.
```

- **Twitter (@turingway):** Used since 2020, our Twitter presence has played an important role in getting the community engaged in real-time with updates and announcements from the project. Follow [@turingway](https://twitter.com/turingway).
- **Fosstodon (@turingway):** Since 2023, we have been using [Fosstodon](https://fosstodon.org/@turingway), a platform for open science practitioners in our community.
- **LinkedIn Page:** Since late 2023, we have been testing and expanding our presence on [LinkedIn](https://www.linkedin.com/company/the-turing-way) among professional networks.

These social media platforms complement our primary communication channels, providing additional avenues (often requiring manual cross-posting) for community engagement and outreach.

## Asynchronous Communications

In addition to our synchronous communication channels, we leverage various platforms for archiving, sharing, and preserving project-related outputs within _The Turing Way_ community, which serve as asynchronous communication channels.

### Monthly Newsletters

> Asynchronous communication is facilitated through {ref}`monthly newsletters<ch-newsletters>`, offering updates on the project's progress, achievements, and upcoming events. These newsletters provide a digestible format for community members to stay informed at their own pace, offering a snapshot of recent developments within _The Turing Way_.

Author: ```suggestion
Asynchronous communication is facilitated through {ref}`monthly newsletters<ch-newsletters>`, offering updates on the project's progress, achievements, and upcoming events. 
These newsletters provide a digestible format for community members to stay informed at their own pace, offering a snapshot of recent developments within _The Turing Way_.
```

### Zenodo Community Page

> _The Turing Way_ community page on [Zenodo](https://zenodo.org/communities/the-turing-way) is the main platform to archive and preserve project-related outputs, including slide decks, reports, datasets (such as the entire GitHub repository and illustrations), and publications. This platform ensures stable and citable records, enhancing accessibility and longevity.

Author: ```suggestion
_The Turing Way_ community page on [Zenodo](https://zenodo.org/communities/the-turing-way) is the main platform to archive and preserve project-related outputs, including slide decks, reports, datasets (such as the entire GitHub repository and illustrations), and publications. 
This platform provides persistent identifiers (DOI) to keep stable and citable records, enhancing accessibility and longevity.
```

### GitHub Repositories

Central to our information storage and sharing, our [GitHub repositories](https://github.com/the-turing-way) serve as the primary hub for project-related documents, resources, and collaborative efforts, offering transparency and collaboration for both synchronous and asynchronous communication.

> 

Author: ```suggestion

Since our transition from a single repository to a GitHub organisation, we are moving towards community-maintained repositories on various projects taking place within the community, including governance work, Working Groups, community events and different kinds of communication outputs.

```

(ch-communication-platforms)=
# Communication Platforms

In _The Turing Way_, our [GitHub organisation](https://github.com/the-turing-way) serve as the primary source of information.
The main [project repository](https://github.com/the-turing-way/the-turing-way) hosts book chapters and project management documentation and resources.

> In addition, the project delivery team, makes use of various communication platforms to facilitate effective information dissemination within and beyond the engaged community.

Reviewer: ```suggestion
In addition the project delivery team makes use of various communication platforms to facilitate effective information dissemination within and beyond the engaged community.
```

While it is not the only communication channel used by the community, it is a popular platform, increasingly used by remote teams around the world.

You can join the workspace using the [joining invitation](https://theturingway.slack.com/join/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw#/shared-invite/email).

### Social media channels

In expanding our reach beyond project-specific communication channels, _The Turing Way_ project delivery team uses social media platforms to connect with a wider audience and share project updates broadly.

- **Twitter (@turingway):** Used since 2020, our Twitter presence has played an important role in getting the community engaged in real-time with updates and announcements from the project. Follow [@turingway](https://twitter.com/turingway).
- **Fosstodon (@turingway):** Since 2023, we have been using [Fosstodon](https://fosstodon.org/@turingway), a platform for open science practitioners in our community.
- **LinkedIn Page:** Since late 2023, we have been testing and expanding our presence on [LinkedIn](https://www.linkedin.com/company/the-turing-way) among professional networks.

> 

Reviewer: Maybe we could add a note here about how a Gitter channel was established, but has since been retired/not been maintained?

Author: Please add that.

Reviewer: Added!

### GitHub Repositories

Central to our information storage and sharing, our [GitHub repositories](https://github.com/the-turing-way) serve as the primary hub for project-related documents, resources, and collaborative efforts, offering transparency and collaboration for both synchronous and asynchronous communication.

Since our transition from a single repository to a GitHub organisation, we are moving towards community-maintained repositories on various projects taking place within the community, including governance work, Working Groups, community events and different kinds of communication outputs.

By February 2024, following repositories host specific working group communications and meeting notes:
- [Accessibility](https://github.com/the-turing-way/accessibility-working-group)
- [Book Dash](https://github.com/the-turing-way/bookdash)
- [Infrastructure](https://github.com/the-turing-way/infrastructure-working-group)
- [Translation and Localisation](https://github.com/TWTranslation)
- [_The Turing Way_ Community Manager and Project Manager Repo](https://github.com/the-turing-way/2024-rcm-rpm-tasks)

> ### _The Turing Way_ Book Chapters

Reviewer: Should the Community Handbook be referenced here?

Author: please do add that in.

The book chapters themselves serve as a form of asynchronous communication, providing an extensive range of practices documented for reuse by others. 
Community members can refer to these chapters at their convenience for onboarding, reference and self-paced learning.
These ultimately serve as an important pathway for the community members to identify where they would like to get involved.

> 

Reviewer: ```suggestion

We also document the practices we use within _The Turing Way_ community in our Community Handbook to provide resources that may help with your community-building efforts, or to learn how to contribute to the project.

```
